jest.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    roots: ["<rootDir>"],
+    testPathIgnorePatterns: ["/node_modules/", "/dist/"],
+    collectCoverageFrom: ["./src/**/*.ts"],
```

Comment:
We should probably enable code coverage here

```suggestion
    collectCoverage: true,
```